frontend/lib/features/home/home_page.dart

```diff
@@ -1,21 +1,169 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_starter/features/home/account_balance.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_starter/features/deposit/deposit_page.dart';
+import 'package:flutter_starter/features/withdraw/withdraw_page.dart';
 import 'package:flutter_starter/l10n/app_localizations.dart';
-import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:flutter_starter/shared/grid.dart';
 
-class HomePage extends HookConsumerWidget {
-  const HomePage({super.key});
+class HomePage extends HookWidget {
+  HomePage({super.key});
+
+  final List<Transaction> txns = [];
 
   @override
-  Widget build(BuildContext context, WidgetRef ref) {
+  Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(title: Text(Loc.of(context).home)),
-      body: const Column(
-        crossAxisAlignment: CrossAxisAlignment.stretch,
+      body: SafeArea(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            buildAccountBalance(context),
+            Padding(
+              padding: const EdgeInsets.symmetric(
+                  horizontal: Grid.side, vertical: Grid.xs),
+              child: Text(
+                Loc.of(context).activity,
+                style: Theme.of(context).textTheme.bodyMedium,
+              ),
+            ),
+            Expanded(
+              child: txns.isEmpty
+                  ? buildEmptyState(context)
```

Comment:
@ethan-tbd I wonder if we should center this in the empty space on the screen vs. at the top. Whatcha think?

Reply:
setting `mainAxisAlignment` to center seems a bit too low? would we have to play with `Align` or just use a bigger`SizedBox`?

![Simulator Screenshot - iPhone 15 Pro Max - 2024-01-31 at 17 21 07](https://github.com/TBD54566975/didpay/assets/125412902/1483c8df-a510-42ad-8e37-4e17ffe9e870)

Reply:
@wesbillman ^ or do you think that's fine as is? been staring at it for too long to tell 😅